Formula/doctl.rb

```diff
@@ -3,8 +3,8 @@
 class Doctl < Formula
   desc "Command-line tool for DigitalOcean"
   homepage "https://github.com/digitalocean/doctl"
-  url "https://github.com/digitalocean/doctl/archive/v1.0.2.tar.gz"
-  sha256 "7a28bd0f9e68eee41761e32267a0d63329c6f0e16e0d8b5afb4a8286b914e98a"
+  url "https://github.com/digitalocean/doctl/archive/v1.1.0.tar.gz"
```

Comment:
This link is now broken: http://bot.brew.sh/job/Homebrew%20Core%20Pull%20Requests/887/version=yosemite/testReport/junit/brew-test-bot/yosemite/install_doctl/